components/lists/FeedList.tsx

```diff
@@ -1,5 +1,13 @@
+import { FontAwesome } from '@expo/vector-icons';
+import { useTheme } from '@react-navigation/native';
+import { Link } from 'expo-router';
 import React from 'react';
-import { View } from 'react-native';
+import { Dimensions, Image, StyleSheet, View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import type { Feed } from '../../global';
+import { Feed as FeedData } from '../../mock/feed';
+import Row from '../common/Row';
+import StyledText from '../common/StyledText';
 
 // TODO: - Implement a feed list-view of feeds as seen in the design
```

Comment:
Once you've implemented the instructions in the task/todos, please remove the comments. this way we are able to easily track what is still pending